build-tools/automation/build.groovy

```diff
@@ -185,7 +197,7 @@ timestamps {
             }
```

Comment:
unrelated to this PR but since I just saw it in a log: `${stageStatus}` should be `${commandStatus}"` in the "run all tests" stage (copy-paste mistake)

Reply:
Let's get that fixed. Commit https://github.com/xamarin/xamarin-android/pull/2797/commits/6442567fc27d31bae1c41832c8917572ef501c87.
Also found that the use of stageStatus versus commandStatus was inconsistent and so aligned all stages to use commandStatus